hibernate-core/src/main/java/org/hibernate/internal/log/ConnectionInfoLogger.java

```diff
@@ -27,70 +28,56 @@
+	ConnectionInfoLogger INSTANCE = Logger.getMessageLogger( ConnectionInfoLogger.class, LOGGER_NAME );
 
-	@LogMessage(level = INFO)
+	@LogMessage(level = DEBUG)
 	@Message(value = "Connection properties: %s", id = 10001001)
 	void connectionProperties(Properties connectionProps);
 
 	@LogMessage(level = WARN)
 	@Message(value = "Using built-in connection pool (not intended for production use)", id = 10001002)
 	void usingHibernateBuiltInConnectionPool();
 
-	@LogMessage(level = INFO)
-	@Message(value = "Autocommit mode: %s", id = 10001003)
-	void autoCommitMode(boolean autocommit);
+	@Message(value = "JDBC Driver class not found: %s", id = 10001003)
+	String jdbcDriverNotFoundErrMsg(String jdbcDriverClass, @Cause Exception e);
 
 	@Message(value = "No JDBC URL specified by property %s", id = 10001004)
 	String jdbcUrlNotSpecified(String property);
 
 	@LogMessage(level = INFO)
-	@Message(value = "Loaded JDBC driver class: %s", id = 10001005)
-	void loadedDriver(String driverClassName);
-
-	@LogMessage(level = INFO)
-	@Message(value = "No JDBC driver class specified by %s", id = 10001010)
-	void noDriver(String property);
-
-	@LogMessage(level = INFO)
-	@Message(value = "Loaded JDBC drivers: %s", id = 10001011)
-	void loadedDrivers(String loadedDrivers);
-
-	@LogMessage(level = INFO)
-	@Message(value = "Connecting with JDBC URL [%s]", id = 10001012)
-	void usingUrl(String url);
+	@Message(value = "Database info:\n%s", id = 10001005)
+	void logConnectionInfoDetails(String databaseConnectionInfo);
 
 	@LogMessage(level = WARN)
 	@Message(id = 10001006, value = "No JDBC Driver class was specified by property `jakarta.persistence.jdbc.driver`, `hibernate.driver` or `javax.persistence.jdbc.driver`")
 	void jdbcDriverNotSpecified();
 
-	@LogMessage(level = INFO)
-	@Message(value = "JDBC isolation level: %s", id = 10001007)
-	void jdbcIsolationLevel(String isolationLevelToString);
-
-	@LogMessage(level = INFO)
+	@LogMessage(level = DEBUG)
 	@Message(value = "Cleaning up connection pool [%s]", id = 10001008)
-	void cleaningUpConnectionPool(String url);
+	void cleaningUpConnectionPool(String info);
 
 	@LogMessage(level = WARN)
 	@Message(value = "Problem closing pooled connection", id = 10001009)
 	void unableToClosePooledConnection(@Cause SQLException e);
 
-	@LogMessage(level = INFO)
-	@Message(value = "Connection pool size: %s (min=%s)", id = 10001115)
-	void hibernateConnectionPoolSize(int poolSize, int minSize);
+	@LogMessage(level = WARN)
+	@Message(value = "Could not destroy connection pool", id = 10001010)
+	void unableToDestroyConnectionPool(@Cause Exception e);
 
 	@LogMessage(level = ERROR)
-	@Message(value = "Error closing connection", id = 10001284)
-	void unableToCloseConnection(@Cause Exception e);
+	@Message(value = "Could not instantiate connection pool", id = 10001011)
+	void unableToInstantiateConnectionPool(@Cause Exception e);
```

Comment:
It's not our usual policy to log errors before re-throwing them.

Perhaps this is such an important error that we should log it, but if so, at minimum I would reduce the level to `DEBUG`.